api/Element.json

```diff
@@ -3017,7 +3017,7 @@
               "notes": "This API was previously available on the <code>Node</code> API."
             },
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
This is wrong, `document.body.prefix` works in Edge 12. Maybe more prototype trouble.